contracts/deposit-contract/sendDepositTx/sendDeposits_test.go

```diff
@@ -0,0 +1,161 @@
+package main
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+	"fmt"
+	"io/ioutil"
+	"testing"
+	"time"
+	"context"
+
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/prysmaticlabs/prysm/beacon-chain/powchain"
+	contracts "github.com/prysmaticlabs/prysm/contracts/deposit-contract"
+	prysmKeyStore "github.com/prysmaticlabs/prysm/shared/keystore"
+	"github.com/prysmaticlabs/prysm/shared/testutil"
+	"github.com/sirupsen/logrus"
+)
+
+func generateValidators(count int64) map[string]*prysmKeyStore.Key {
+	validatorKeys := make(map[string]*prysmKeyStore.Key)
+	for i := int64(0); i < count; i++ {
+		validatorKey, err := prysmKeyStore.NewKey(rand.Reader)
+		if err != nil {
+			log.Errorf("Could not generate random key: %v", err)
+		}
+		validatorKeys[hex.EncodeToString(validatorKey.PublicKey.Marshal())] = validatorKey
+	}
+	return validatorKeys 
+
+}
+
+//should I explicitly return err here?
+func sendDeposits(testAcc *contracts.TestAccount, validatorKeys map[string]*prysmKeyStore.Key,
+	numberOfDeposits int64) {
+	depositAmountInGwei := uint64(contracts.Amount32Eth.Int64())
```

Comment:
I would like to convert contracts.Amount32Eth ( *big.Int) to depositAmountInGwei (uint64). I googled it several times, found no answer. @nisdas, can you guide me please, what is the best way to do it?

Reply:
https://golang.org/pkg/math/big/#Int.Uint64